com.ibm.streamsx.topology/opt/python/packages/streamsx/rest_primitives.py

```diff
@@ -395,6 +400,47 @@ class Job(_ResourceElement):
         >>> print (jobs[0].health)
         healthy
     """
+    def get_application_trace(self, path=None, name=None):
+        """Retrieves the application log and saves it to the specified path with the given name as a tar file.
+
+        If logs are retrieved with the same path and name as previously retrieved logs, the prior logs will be
+        overwritten.
+
+        Attributes:
+            path (str): a valid directory in which to save the application log output. Defaults to current dir.
+            name (str): the filename of the created tar file. Defaults to a name based on the job name.
```

Comment:
The code doesn't create a file with `name` when it is specified, it appends `_app_logs.tar`.

Reply:
Updated to `prefix` from `name`.